crates/telio-lana/src/lib.rs

```diff
@@ -115,3 +140,50 @@ pub fn deinit_lana() -> Result<moose::Result, moose::Error> {
 pub fn is_lana_initialized() -> bool {
     MOOSE_INITIALIZED.load(DEFAULT_ORDERING)
 }
+
+#[cfg(test)]
+mod test {
```

Comment:
Please add two more tests. One to verify behaviour when `moose::init` fails and a second one to verify behaviour when the `moose::init` doesn't call `init_cb`.

Reply:
In fact it should be 3 tests, since `init_cb` can either return an error or no be called at all.

Reply:
Added